examples/circuitplayground_light_neopixels.py

```diff
@@ -13,7 +13,7 @@
 
 
 def scale_range(value):
-    """Scale a value from 0-320 (light range) to 0-10 (the number of NeoPixels).
+    """Scale a value from 0-320 (light range) to 0-9 (NeoPixel range).
     Allows remapping light value to pixel position."""
     return int(value / 320 * 10)
```

Comment:
If the value is 320 here, it will return 10. However, we are unlikely to hit it. If we changed this to `round(value / 320 * 9)`, I think that would fix it.

Reply:
Good catch. Updated!